config_defaults/subtests/docker_cli/run.ini

```diff
@@ -54,6 +54,20 @@ last_name_sticks = yes
 names_count = 1000
 exit_status = 0
 
+[docker_cli/run/run_tmpfs]
+cmd = 'sleep 5s'
+# command to execute in running container
+# passed as argument to exec
+fndmnt = findmnt -n %(tmpfs_path)s 
+# Test will find container by CID instead of name, CID must appear
+# on stdout.
+run_options_csv = --detach=true,--tmpfs %(tmpfs_path)s
+tmpfs_path = /run
```

Comment:
$ pydoc ConfigParser | grep -A1 resolve  
        would resolve the "%(dir)s" to the value of dir.  All reference
        expansions are done late, on demand.

I had to look it up before I commented :-). And the fact that I had to look it up is itself an argument for declaring it early.